src/components/Tooltip/BaseTooltip.js

```diff
@@ -43,6 +76,14 @@ function Tooltip(props) {
     const isAnimationCanceled = useRef(false);
     const prevText = usePrevious(text);
 
+    const target = useRef(null);
+    const initialMousePosition = useRef({x: 0, y: 0});
+
+    const updateTargetAndMousePosition = useCallback((e) => {
+        target.current = e.target;
```

Comment:
We should have used the `currentTarget` (the element that is listening to the mouse enter event) instead of `target` (the element that caused the event to fire and bubble up). Coming from https://github.com/Expensify/App/issues/29678